pkg/kubelet/dockertools/manager.go

```diff
@@ -979,11 +979,26 @@ func (dm *DockerManager) Version() (kubecontainer.Version, error) {
 		return nil, fmt.Errorf("docker: failed to get docker version: %v", err)
 	}
 
+	engineVersion := env.Get("Version")
+	version, err := docker.NewAPIVersion(engineVersion)
+	if err != nil {
+		glog.Errorf("docker: failed to parse docker server version %q: %v", engineVersion, err)
+		return nil, fmt.Errorf("docker: failed to parse docker server version %q: %v", engineVersion, err)
+	}
+	return dockerVersion(version), nil
+}
+
+func (dm *DockerManager) APIVersion() (kubecontainer.Version, error) {
```

Comment:
Can we instead update the `container.Version` interface to provide both API and engine version?